make_tarball.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -eu
+
+if [ "$#" -ne "2" ]; then
+    echo "Usage: $0 NAME TARGET"
+    echo
+    echo "Creates a file NAME.tar.gz containing the dune binary distro."
+    echo "TARGET is the nix target that gets built. It should be one of:"
+    echo "  - .#dune.dynamic (builds a dynamically-linked dune executable)"
+    echo "  - .#dune.static (builds a statically-linked dune executable)"
+    exit 1
```

Comment:
For my own edification, where is the logic that will actually make this decision based on the value of `TARGET`? Searching this repo for those strings didn't seem to turn anything up for me.

Reply:
Oh yes. That I saw. Perhaps that intended usage shoukd be noted here then? There is nothing about this script itself as is that lets readers of the file know it has a coupling like that with the GitHub workflow.